test/expected/05-create_shards.out

```diff
@@ -227,27 +238,71 @@ WARNING:  could not create shard on "adeadhost:5432"
 \set VERBOSITY default
 SELECT storage, min_value, max_value FROM pgs_distribution_metadata.shard
 	WHERE relation_id = 'foreign_table_to_distribute'::regclass
-	ORDER BY (min_value COLLATE "C") ASC;
+	ORDER BY (min_value::integer) ASC;
  storage |  min_value  |  max_value  
 ---------+-------------+-------------
- f       | -1073741828 | -805306374
- f       | -1342177283 | -1073741829
- f       | -1610612738 | -1342177284
- f       | -1879048193 | -1610612739
- f       | -2147483648 | -1879048194
- f       | -268435463  | -9
- f       | -536870918  | -268435464
- f       | -8          | 268435446
- f       | -805306373  | -536870919
- f       | 1073741812  | 1342177266
- f       | 1342177267  | 1610612721
- f       | 1610612722  | 1879048176
- f       | 1879048177  | 2147483647
- f       | 268435447   | 536870901
- f       | 536870902   | 805306356
- f       | 805306357   | 1073741811
+ f       | -2147483648 | -1879048193
+ f       | -1879048192 | -1610612737
+ f       | -1610612736 | -1342177281
+ f       | -1342177280 | -1073741825
+ f       | -1073741824 | -805306369
+ f       | -805306368  | -536870913
+ f       | -536870912  | -268435457
+ f       | -268435456  | -1
+ f       | 0           | 268435455
+ f       | 268435456   | 536870911
+ f       | 536870912   | 805306367
+ f       | 805306368   | 1073741823
+ f       | 1073741824  | 1342177279
+ f       | 1342177280  | 1610612735
+ f       | 1610612736  | 1879048191
+ f       | 1879048192  | 2147483647
 (16 rows)
 
+-- test shard creation using weird shard count
+CREATE TABLE weird_shard_count
+(
+	name text,
+	id bigint
+);
+\set VERBOSITY terse
+SELECT master_create_distributed_table('weird_shard_count', 'id');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT master_create_worker_shards('weird_shard_count', 7, 1);
+WARNING:  Connection failed to adeadhost:5432
+WARNING:  could not create shard on "adeadhost:5432"
+WARNING:  Connection failed to adeadhost:5432
+WARNING:  could not create shard on "adeadhost:5432"
+WARNING:  Connection failed to adeadhost:5432
+WARNING:  could not create shard on "adeadhost:5432"
+WARNING:  Connection failed to adeadhost:5432
+WARNING:  could not create shard on "adeadhost:5432"
+ master_create_worker_shards 
+-----------------------------
+ 
+(1 row)
+
+\set VERBOSITY default
+-- pg_shard ensures all shards are roughly the same size
+SELECT max_value::integer-min_value::integer AS shard_size
```

Comment:
should we add white space: `max_value::integer-min_value::integer AS shard_size` =>
  `max_value::integer - min_value::integer AS shard_size`